src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -861,179 +901,180 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
+                            ShowSevereError(state,
+                                            format("{}{}=\"{}\", invalid flow/area specification [<0.0]={:.3R}",
+                                                   RoutineName,
+                                                   cCurrentModuleObject,
+                                                   thisInfiltration.Name,
+                                                   rNumericArgs(2)));
+                            ErrorsFound = true;
                         }
-                        if (lAlphaFieldBlanks(3)) {
-                            ShowWarningError(state,
-                                             std::string{RoutineName} + cCurrentModuleObject + "=\"" + state.dataHeatBal->Infiltration(Loop).Name +
-                                                 "\", " + cAlphaFieldNames(4) + " specifies " + cNumericFieldNames(3) +
-                                                 ", but that field is blank.  0 Infiltration will result.");
-                        }
-                    } else if (SELECT_CASE_var == "FLOW/EXTERIORWALLAREA") {
-                        if (state.dataHeatBal->Infiltration(Loop).ZonePtr != 0) {
-                            if (rNumericArgs(3) >= 0.0) {
-                                state.dataHeatBal->Infiltration(Loop).DesignLevel =
-                                    rNumericArgs(3) * state.dataHeatBal->Zone(state.dataHeatBal->Infiltration(Loop).ZonePtr).ExtGrossWallArea;
-                                if (state.dataHeatBal->Zone(state.dataHeatBal->Infiltration(Loop).ZonePtr).ExtGrossWallArea <= 0.0) {
-                                    ShowWarningError(state,
-                                                     std::string{RoutineName} + cCurrentModuleObject + "=\"" +
-                                                         state.dataHeatBal->Infiltration(Loop).Name + "\", " + cAlphaFieldNames(4) + " specifies " +
-                                                         cNumericFieldNames(3) + ", but Exterior Wall Area = 0.  0 Infiltration will result.");
-                                }
-                            } else {
-                                ShowSevereError(state,
-                                                format("{}{} = \"{}\", invalid flow/exteriorwallarea specification [<0.0]={:.3R}",
-                                                       RoutineName,
-                                                       cCurrentModuleObject,
-                                                       state.dataHeatBal->Infiltration(Loop).Name,
-                                                       rNumericArgs(3)));
-                                ErrorsFound = true;
+                    }
+                    if (lAlphaFieldBlanks(2)) {
+                        ShowWarningError(state,
+                                         std::string{RoutineName} + cCurrentModuleObject + "=\"" + thisInfiltration.Name + "\", " +
```

Comment:
Maybe search for `std::string{`

Reply:
Reformatted all `std::string{` and also searched for instances of string concatenation with `" + `.